src/Hyperion/Main.hs

```diff
@@ -78,7 +81,8 @@ options = do
          (Options.strOption
             (Options.long "output" <>
              Options.short 'o' <>
-             Options.metavar "FILENAME"))
+             Options.help "Outputs the benchark to the directory" <>
```

Comment:
It actually has to be a directory because we have no way to know whether the user gives us a directory or a file (if they don't already exist).